src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx).
```

Comment:
Thoughts on leaving the old migration as a hosted page instead of a GitHub URL, and just unlisting it from the sidebar? Something like `upgrade-guide-v1.md`?

Reply:
We are just waiting for @delucis to report back on how this might affect our site search results before making a final decision re: placement of older guides.